pcdsdevices/tests/conftest.py

```diff
@@ -135,9 +138,13 @@ def sort_key(cls):
     return list(sorted(set(devices), key=sort_key))
 
 
-def find_all_device_classes() -> list:
-    """Find all device classes in pcdsdevices and return them as a list."""
-    return find_all_classes(ophyd.Device)
+def find_all_device_classes(skip: Optional[List[str]] = None) -> list:
```

Comment:
Another tiny tiny nitpick: pick either `List` or `list` for your annotation and stick to it imo